examples/deployment/README.md

```diff
@@ -41,15 +46,18 @@ curl localhost:8091/metrics
 
 ## Cloud Deployments
 
-For better persistence and performance you may want to run in your datacenter or a cloud.  Here are some simple cloud deployment templates:
+For better persistence and performance you may want to run in your datacenter or
+a cloud. Here are some simple cloud deployment templates:
 
 ### Run in GCP
 
 TODO
 
 ### Run in AWS
 
-With a pair of AWS keys [accessible to Terraform](https://www.terraform.io/docs/providers/aws/), this template deploys a simple Trillian setup in AWS using EC2 and RDS MySQL.
+With a pair of AWS keys
+[accessible to Terraform](https://www.terraform.io/docs/providers/aws/), this
+template deploys a simple Trillian setup in AWS using EC2 and RDS MySQL.
```

Comment:
supernit: Maybe put a semicolon in the end, like in all such paragraphs above? Alternatively, use period everywhere?

Reply:
Done.